src/converse-register.js

```diff
@@ -390,6 +416,10 @@
                     if (ev && ev.preventDefault) { ev.preventDefault(); }
                     _converse.connection.reset();
                     this.render();
+                    if(_converse.registration_domain !== '')
+                    {
```

Comment:
Why do you start the curly bracket on a new line? Please stick to the coding convention used throughout. Also, add spaces between your `if` statements and the first left bracket `(`